GitHubCore.ps1

```diff
@@ -153,7 +153,14 @@ function Invoke-GHRestMethod
 
     $hostName = $(Get-GitHubConfiguration -Name "ApiHostName")
 
-    $url = "https://api.$hostName/$UriFragment"
+    if ($hostName -eq "github.com")
```

Comment:
I have generally been avoiding double-quoted strings unless I am expecting for them to be interpreted due to embedded expressions/variables/etc...

```suggestion
    if ($hostName -eq 'github.com')
```

Reply:
:) sure...i will change to single